lib/hardwareWalletUtilities.js

```diff
@@ -0,0 +1,106 @@
+import { Script, Outpoint, TX, MTX, Coin } from 'bcoin';
+import { LedgerBcoin, U2F, LedgerTXInput } from 'bledger/lib/bledger-browser';
```

Comment:
I feel like the `bledger-browser` decision should maybe be done via a webpack config, similar to how we do it for bcoin in the main `bpanel` repo since we will always want to use the browser references.

Reply:
I guess we already do that for `bcoin` but then it makes our code depend on a config file which then people will have to use, its a hidden dependency and I like to minimize those

Reply:
Yeah, that sounds good. Since we don't have a webpack config for this package and bpanel doesn't need to have any awareness at all of bledger, this makes sense.